subgraph/src/fleek-nfa.ts

```diff
@@ -97,6 +145,13 @@ export function handleCollectionRoleRevoked(
   entity.transactionHash = event.transaction.hash;
 
   entity.save();
+
+  if (event.params.role === 0) {
```

Comment:
Right now we don't call this anywhere. But something to think about in the future, if we do a transfer at the collection level, which may create both the grant and revoke roles so that means both handlers will run. We should just make sure it doesn't get overwritten or deleted or anything.

Reply:
Good point. I'll have to check if a scenario like that will result in overriding or not.